assembl/tests/views_tests/test_graphql_user.py

```diff
@@ -385,3 +385,13 @@ def test_graphql_delete_user_with_social_account(graphql_request, participant1_u
     })
     assert res.errors is None
     assert len(participant1_user.social_accounts) == 0
+
+
+def test_graphql_delete_user_with_configurable_fields(graphql_request, participant1_user, profile_field_for_participant_user):
+    """
+    Testing deletion of user with extra configurable text fields
+    """
+    res = schema.execute(DELETE_USER_INFORMATION_MUTATION, context_value=graphql_request, variable_values={
+        "id": to_global_id('AgentProfile', participant1_user.id)
+    })
```

Comment:
You don't test anything here. You should do the sql query to check you don't have profile fields anymore.